get rid of the following warning using perl 5.10.0:
v-string in use/require non-portable at /usr/lib/perl5/vendor_perl/5.10.0/i386-linux-thread-multi/XML/LibXML/Reader.pm line 11.

diff -p -up XML-LibXML-1.65/lib/XML/LibXML/Reader.pm.pix XML-LibXML-1.65/lib/XML/LibXML/Reader.pm
--- XML-LibXML-1.65/lib/XML/LibXML/Reader.pm.pix	2007-09-25 16:18:30.000000000 +0000
+++ XML-LibXML-1.65/lib/XML/LibXML/Reader.pm	2008-01-17 12:58:21.000000000 +0000
@@ -2,7 +2,6 @@ package XML::LibXML::Reader;
 use XML::LibXML;
 
 use strict;
-use warnings;
 
 use vars qw ($VERSION);
 $VERSION = "1.65"; # VERSION TEMPLATE: DO NOT CHANGE
